src/modules/avformat/producer_avformat.c

```diff
@@ -2092,10 +2097,7 @@ static int producer_get_image( mlt_frame frame, uint8_t **buffer, mlt_image_form
 				(codec_params->field_order == AV_FIELD_PROGRESSIVE ||
 				 codec_params->field_order == AV_FIELD_UNKNOWN) );
 	}
-	av_frame_unref( self->video_frame );
```

Comment:
Was leaving this out intentional, or is it just missing above? Without this, the memory leak addressed [here](https://github.com/mltframework/mlt/pull/621#issuecomment-732147656) comes back.

Reply:
intentional, read above. video_frame must be reusable. We need to find a fix for the memory leak some other way.